pkg/plugins/generic/generic_plugin.go

```diff
@@ -197,3 +203,24 @@ func isCommandNotFound(err error) bool {
 	}
 	return false
 }
+
+func handleSriovModeStateChange(iface *sriovnetworkv1.Interface, old *sriovnetworkv1.SriovNetworkNodeState) (needDrain bool, err error) {
+	err = nil
+	needDrain = false
+	if iface.SriovMode == "" {
```

Comment:
if iface.SriovMode **!=** "" ?

Reply:
by mistake it should be `if iface.SriovMode != ""`